chcemvediet/apps/__init__.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+import django.db.backends.sqlite3.base
+from django.apps import AppConfig
+from django.db.backends import utils as backend_utils
+
+
+class ChcemvedietConfig(AppConfig):
+    name = u'chcemvediet.apps'
+
+    def ready(self):
+        django.db.backends.sqlite3.base._sqlite_format_dtdelta = self._sqlite_format_dtdelta
+
+    # fixed in django 3.1rc1
```

Comment:
Zislo by sa trochu podrobnejsie vysvetlenie o co ide. Napis sem pls strucny suhrn z tohto komentara https://github.com/martinmacko47/chcemvediet/pull/290#discussion_r449566566

Komentar zacni "Workarround for bug in `_sqlite_format_dtdelta`." a potom vysvetlenie v com je problem. Aby ked si ten komentar precitas o dva roky, tak ti z neho bolo jasne, preco tu ten patch mame.

Reply:
:ok: